tests/integration/test_engines.py

```diff
@@ -0,0 +1,85 @@
+from os import getenv
+from searx import settings, engines, settings
+from searx.search import SearchQuery, Search, EngineRef, initialize
+from tests import SearxTestCase
+from typing import Tuple, Optional
+import sys
+import logging
+from flask import Flask
+from dotenv import load_dotenv, find_dotenv
+
+# Root directeory .env.test file
+load_dotenv(find_dotenv("../../.env.test", raise_error_if_not_found=False))
+
+logger = logging.getLogger('integration-test')
+logger.level = logging.INFO
+stream_handler = logging.StreamHandler(sys.stdout)
+logger.addHandler(stream_handler)
+
+
+def test_single_engine(app: Flask, engine_name: str) -> Tuple[str, Optional[Exception], int]:
+    logger.debug(f'Testing Engine: {engine_name}')
+    try:
+        with app.test_request_context():
+            # test your app context code
+            search_query = SearchQuery(
+                'test', [EngineRef(engine_name, 'general')], 'en-US', SAFESEARCH, PAGENO, None, None
```

Comment:
can configure the search term in an environment variable - can do so later if PR concept is OK